docs.css

```diff
@@ -128,15 +135,16 @@ header{
     min-height: 8rem;
     height: auto;
     margin-left: 1rem;
-    padding: 1rem 1rem 1rem 1rem;
+    padding: 1rem;
 }
 
 .comp-container-content-row,
 .image-container,
 .avatar-container,
 .card-container,
 .card-h-container-sm,
-.card-h-container-lg{
+.card-h-container-lg,
+.list-container{
```

Comment:
same here, `.container` could have been a base class.

Reply:
acknowledged. I have changed this now and yes the code looks cleaner !

Reply:
Thank you very much for the review @venkateshwarans venkateshwarans